nc-games-fe/src/Components/Review-Comments.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState, useEffect } from "react";
+import { fetchReviewComments } from "./utils";
+
+const ReviewComments = ({ reviewId }) => {
+  const [comments, setComments] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [hasComments, setHasComments] = useState(false);
+
+  useEffect(() => {
+    fetchReviewComments(reviewId)
+      .then((data) => {
+        setComments(data.comments);
+        setIsLoading(false);
+        setHasComments(data.comments.length > 0);
+      });
```

Comment:
This is interesting, you have three set state functions within this single .then, but something to consider is that these will not wait for each other to finish so there is a chance that your setIsLoading state will be set to false before you populate your comments state and hasComments state.